portfolio/src/main/webapp/script.js

```diff
@@ -81,4 +79,18 @@ function createListElement(text) {
   var textNode = document.createTextNode(text);
   pElement.appendChild(textNode);
   return pElement;
+}
+
+/**
+ * Function to check if the user is logged In.
+ */
+async function isUserLoggedIn() {
```

Comment:
In general, a function with a name like `is<somethingSomething>` is expected to just return a boolean. It would make a lot of sense to write something like:

```
if (isUserLoggedIn()) {
    displayUserInfo();
} else {
    displayLoginLink();
}
```

So if your function is doing something else (in this case, updating the contents of the "login" div), you might want to rename it to something more expressive of what it's doing (e.g. `updateLoginElement`).

Reply:
Thanks for the correction. Will change that for the next commit.